flink-table/flink-table-common/src/test/java/org/apache/flink/table/api/TableSchemaTest.java

```diff
@@ -328,7 +328,7 @@ void testPrimaryKeyPrinting() {
                                 + " |-- f0: BIGINT NOT NULL\n"
                                 + " |-- f1: STRING NOT NULL\n"
                                 + " |-- f2: DOUBLE NOT NULL\n"
-                                + " |-- CONSTRAINT pk PRIMARY KEY (f0, f2)\n");
+                                + " |-- CONSTRAINT `pk` PRIMARY KEY (`f0`, `f2`) NOT ENFORCED\n");
```

Comment:
Why this changed ?

Reply:
there are some minor differences between the new and old APIs
```java   
public final String asSummaryString() {
        return String.format(
                "CONSTRAINT %s %s (%s)%s",
                EncodingUtils.escapeIdentifier(getName()),
                getTypeString(),
                columns.stream()
                        .map(EncodingUtils::escapeIdentifier)
                        .collect(Collectors.joining(", ")),
                isEnforced() ? "" : " NOT ENFORCED");
  }
```
 The method `escapeIdentifier` of the new api adds a symbol `` to each field
```java
    public static String escapeIdentifier(String s) {
        return "`" + escapeBackticks(s) + "`";
    }
```

Reply:
this is the deprecated UniqueConstraint 
```java
    public final String asSummaryString() {
        final String typeString;
        switch (getType()) {
            case PRIMARY_KEY:
                typeString = "PRIMARY KEY";
                break;
            case UNIQUE_KEY:
                typeString = "UNIQUE";
                break;
            default:
                throw new IllegalStateException("Unknown key type: " + getType());
        }

        return String.format(
                "CONSTRAINT %s %s (%s)", getName(), typeString, String.join(", ", columns));
    }
```